x-pack/test/functional/fixtures/kbn_archiver/saved_objects_management/spaces_integration_space_1.json

```diff
@@ -0,0 +1,27 @@
+{
+  "attributes": {
```

Comment:
Should this file be removed?  I don't think it's being used.

Reply:
argh! I thought I dropped it!  On it.